web/src/personal_menu_popover.js

```diff
@@ -61,9 +61,8 @@ export function initialize() {
 
             $popper.one("click", ".personal-menu-actions .view_full_user_profile", (e) => {
                 const user_id = elem_to_user_id($(e.target).closest(".personal-menu-actions"));
-                const user = people.get_by_user_id(user_id);
                 popovers.hide_all();
-                user_profile.show_user_profile(user);
+                browser_history.go_to_location(`user/${user_id}`);
                 e.preventDefault();
```

Comment:
We don't need a click handler here now, we can just generate the link at the time of rendering the user profile popover use it in the `href` of `a` tag of the element.

Reply:
I have removed the click handler and generated the link using href as suggested.